### ISSUE TITLE
采用如下配置训练svtrvv2算法，数据是转换好的lmdb格式，训练几个epoch后，就出现过拟合了，老是这样，请官方告诉我这个为啥？

### ISSUE THREAD

**Issue by wangyuangen615** [Topdu/OpenOCR#153]
采用如下配置训练svtrv2算法，数据是转换好的lmdb格式；采用的是断点训练方式，第一次设置的学习率是0.00012，weight_decay: 0.05，后面出现过拟合后，改成学习率是0.00024，weight_decay: 0.08；训练的config.yml(svtrv2_rctc.yml)的配置参数如下所示：
Global:
  device: gpu
  epoch_num: 80
  log_smooth_window: 20
  print_batch_step: 10
  output_dir: ./output/finetune/svtrv2_rctc_finetune/
  save_epoch_step:
  - 15
  - 1
  eval_epoch_step:
  - 0
  - 1
  eval_batch_step:
  - 0
  - 11210
  cal_metric_during_train: true
  pretrained_model: null
  checkpoints: ./output/finetune/svtrv2_rctc_finetune/latest.pth
  use_tensorboard: false
  infer_img: null
  character_dict_path: /data/ocr_dataset/ch_dataset/dataset_dic/sum6213_result_dic.txt
  max_text_length: 100
  use_space_char: false
  save_res_path: ./output/rec/u14m_filter/predicts_svtrv2_rctc.txt
  use_amp: true
  distributed: true
Optimizer:
  name: AdamW
  lr: 0.00024
  weight_decay: 0.08
  filter_bias_and_bn: true
LRScheduler:
  name: OneCycleLR
  warmup_epoch: 4
  cycle_momentum: false
Architecture:
  model_type: rec
  algorithm: SVTRv2
  Transform: null
  Encoder:
    name: SVTRv2LNConvTwo33
    use_pos_embed: false
    dims:
    - 128
    - 256
    - 384
    depths:
    - 6
    - 6
    - 6
    num_heads:
    - 4
    - 8
    - 12
    mixer:
    - - Conv
      - Conv
      - Conv
      - Conv
      - Conv
      - Conv
    - - Conv
      - Conv
      - FGlobal
      - Global
      - Global
      - Global
    - - Global
      - Global
      - Global
      - Global
      - Global
      - Global
    local_k:
    - - 5
      - 5
    - - 5
      - 5
    - - -1
      - -1
    sub_k:
    - - 1
      - 1
    - - 2
      - 1
    - - -1
      - -1
    last_stage: false
    feat2d: true
  Decoder:
    name: RCTCDecoder
Loss:
  name: CTCLoss
  zero_infinity: true
PostProcess:
  name: CTCLabelDecode
Metric:
  name: RecMetric
  main_indicator: acc
  is_filter: true
Train:
  dataset:
    name: RatioDataSetTVResize
    ds_width: true
    padding: false
    data_dir_list:
    - /data/ocr_dataset/ch_dataset/ch_scentence_lmdb/train/train_HYT20-23cn_data_mdb
    - /data/ocr_dataset/ch_dataset/ch_scentence_lmdb/train/chinese_train_lmdb
    transforms:
    - DecodeImagePIL:
        img_mode: RGB
    - PARSeqAugPIL: null
    - CTCLabelEncode:
        character_dict_path: /data/ocr_dataset/ch_dataset/dataset_dic/sum6213_result_dic.txt
        use_space_char: false
        max_text_length: 100
    - KeepKeys:
        keep_keys:
        - image
        - label
        - length
  sampler:
    name: RatioSampler
    scales:
    - - 640
      - 48
    first_bs: 48
    fix_bs: false
    divided_factor:
    - 4
    - 16
    is_training: true
  loader:
    shuffle: true
    batch_size_per_card: 48
    drop_last: true
    max_ratio: 16
    num_workers: 8
Eval:
  dataset:
    name: RatioDataSetTVResize
    ds_width: true
    padding: false
    data_dir_list:
    - /data/ocr_dataset/ch_dataset/ch_scentence_lmdb/val/val_HYT20-23cn_data_mdb
    - /data/ocr_dataset/ch_dataset/ch_scentence_lmdb/val/chinese_val_lmdb
    transforms:
    - DecodeImagePIL:
        img_mode: RGB
    - CTCLabelEncode:
        character_dict_path: /data/ocr_dataset/ch_dataset/dataset_dic/sum6213_result_dic.txt
        use_space_char: false
        max_text_length: 100
    - KeepKeys:
        keep_keys:
        - image
        - label
        - length
  sampler:
    name: RatioSampler
    scales:
    - - 640
      - 48
    first_bs: 48
    fix_bs: false
    divided_factor:
    - 4
    - 16
    is_training: false
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 48
    max_ratio: 16
    num_workers: 8
filename: svtrv2_rctc
config: configs/rec/svtrv2/svtrv2_rctc.yml
local_rank: '1'
eval: true

训练几个epoch后，就出现过拟合了，老是这样，请官方告诉我这个为啥？如下图所示：
<img width="1699" height="898" alt="Image" src="https://github.com/user-attachments/assets/f8498d03-5914-4a7c-9d84-f361982f27d9" />

**Comment by super-tian** [comment]
这不表明训练的挺好？

**Comment by wangyuangen615** [comment]
才进行到第4个epoch，acc就是1了，到了20个epoch的时候，我拿这个训练的模型，使用测试数据跑了一遍，字符的准确率才95%-96.23%，字段准确率是65%-78%左右，你这个差距太大了吧，这个就是过拟合呀
